dcc-storage-client/src/main/java/org/icgc/dcc/storage/client/config/ClientConfig.java

```diff
@@ -214,13 +222,20 @@ private void configureSystemHttpTimeouts() {
         Long.toString(properties.getReadTimeoutSeconds() * 1000));
   }
 
+  /**
+   * Populates Authorization header with OAuth token
+   * @param HttpClientBuilder instance to set Default Header on. Existing Headers will be overwritten.
+   */
   private void configureOAuth(HttpClientBuilder client) {
     val accessToken = properties.getAccessToken();
 
-    val defined = accessToken != null;
+    val defined = (accessToken != null) && (!accessToken.isEmpty());
```

Comment:
Guava `Strings`

`isNullOrEmpty(accessToken)`

But nice use of naming variable :)